devicetypes/smartthings/nyce-motion-sensor.src/nyce-motion-sensor.groovy

```diff
@@ -13,6 +13,7 @@
  *  for the specific language governing permissions and limitations under the License.
  *
  */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
```

Comment:
While I was working on the change to expose this code to the DTHs I talked with Luke about the possibility of making it auto imported, but we decided that the change was complicated and not worth the gains to do so.  There are also other DTHs in SmartThingsPublic that similarly do imports for things needed in the DTH.  See zwave-lock for example.